src/FormGroup.js

```diff
@@ -32,6 +32,7 @@ const FormGroup = (props) => {
 
   const classes = mapToCssModules(classNames(
     className,
+    'position-relative',
```

Comment:
wouldn't be better to give user control over `position-relative` ?  
If he adds tooltips he can add that class himself!?!